README.md

```diff
@@ -27,10 +27,15 @@ repositories {
 }
 
 dependencies {
-    'com.basecamp:turbolinks:1.0.2'
+    compile 'com.basecamp:turbolinks:1.0.2'
 }
 ```
 
+Add the INTERNET permission to `AndroidManifest.xml`
```

Comment:
How about moving this to the "prerequisites" section right below. That would make sense. Would also be good to clarify this is [a permission required by Android for all WebViews](https://developer.android.com/reference/android/webkit/WebView.html), not something specific to the library. 

Thanks!